06_puppy/jimbotech/puppy_test.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+)
+
+func TestMain2(t *testing.T) {
```

Comment:
I wonder what ever happened to `TestMain1` ? Perhaps this could take its place, and even be called `TestMain`?

Reply:
It is now, done